src/Markdown.MAML/Parser/MarkdownParser.cs

```diff
@@ -148,7 +148,15 @@ private void InitializePatternList()
 
                         {   "italic",
                             @"\*(.+?)\*",
-                            this.CreateItalicSpan }
+                            this.CreateItalicSpan },
+
+                        {   "bold2",
+                            @"(?<![a-zA-Z0-9])__(.+?)__(?![a-zA-Z0-9])",
```

Comment:
@vors I think I see what you mean. I need to look into it further and see if I can come up with an alternate way to accomplish the same.